src/main.rs

```diff
@@ -496,7 +517,13 @@ fn print_avg_colored(
     write!(t, "Package ").expect("term::write failed");
     write_with_colours(t, pkg, options, None, Some(term::Attr::Bold));
     // Normal "is affected by {issues}"
-    write!(t, " is affected by {}. ", avg.issues.join(", ")).expect("term::write failed");
+    write!(
+        t,
+        " is affected by {}: {}. ",
+        avg.issues.join(", "),
+        avg.avg_type
```

Comment:
I wouldn't show the types by default since `arch-audit`'s output is already too much verbose for me; maybe it's time to add a `--verbose` flag and then show the types when that flag is set.

Reply:
Removed it from `print_avg_colored`. You can still print it using the `--format` flag.

Reply:
Maybe instead of printing the CVE numbers by default, the type should be printed?

For the average user, the type of the vulnerability is more important  than the CVE number.

What do you think?

Reply:
I think you've got a point. Then please always show the types and add a `-c, --show-cve` flag to show the CVEs.

Thank you!

Reply:
done